fix: copy a role with the caller's session (#141)
Jahia/rolesmanager: .chachalog/Q3BlNJhm.md

---
rolesmanager: patch
---

> Copying a role now runs on the caller session, the same way creating one already does, so both paths ask the operator for the same rights. The identifier submitted with the copy must designate a role stored under /roles; anything else is refused with a message.

Author: This fragment is the most durable public text this PR produces — chachalog fragments land in the release notes verbatim, `a7Fk2pLm.md` is the body of the 8.9.0 release word for word. Worth writing it as the contract the module offers, not as a comparison with what it offered before.

- `now runs`, `the same way creating one already does`, and `so both paths ask the operator for the same rights` are all statements about the *previous* behavior; a reader reconstructs it from them without the diff.
- The second sentence already carries the whole user-facing fact on its own — it states the rule the identifier has to satisfy.

**The suggestion below keeps both facts and drops the contrast.** The PR description opens the same way and is worth the same pass before this leaves draft.

```suggestion
Copying a role requires the same rights as creating one, and the identifier submitted with the copy must designate a role stored under /roles; anything else is refused with a message.
```

Author: Fixed in 49b7ba6 — the fragment now states the contract in a single sentence. Applied the same pass to the PR description, which opened the same way.
